scipy/sparse/csgraph/_tools.pyx

```diff
@@ -564,6 +564,11 @@ def construct_dist_matrix(graph,
     point i to point j.  If no path exists between point i and j, then
     predecessors[i, j] = -9999
 
+    It should be noted that `shortest_path` also returns distance matrix
```

Comment:
Can you change the previous paragraph's first line to be something like:
```
The predecessor matrix is of the form optionally returned by
```
That word "optionally" could make a large difference to a reader scanning quickly.

Then this new paragraph can/should replace "also" with "only".